wiki/migrations/0004_increase_slug_size.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wiki', '0003_ip_address_conv'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='urlpath',
+            name='slug',
+            field=models.SlugField(max_length=255, null=True, verbose_name='slug', blank=True),
+        ),
```

Comment:
@edx/devops This migration is altering a field on `wiki_urlpath`. Inspecting the read-replica, that will touch ~13.5k rows. Any concerns about locking that table when this migration runs?

```
efischer19@ip-10-254-3-176 i-e09f781d:~$ /edx/bin/prod-edx-edxapp-mysql.sh
...
mysql> select count(*) from wiki_urlpath;
+----------+
| count(*) |
+----------+
|    13465 |
+----------+
1 row in set (0.01 sec)
```